### PR TITLE
Implement table cell updates: start with bool: update module enabled

### PR DESCRIPTION
**Pull request by ewoudwijma**
index.css: modal window using gradient colors

index.js: 
- add # in table cells (rowNr)
- add disabled to checkbox, button and range if ro
- processVarNode: use hasOwnProperty to catch boolean false also
- processVarNode: add isArray to process array values (currently checkbox only supported -> WIP)

pio.ini: disable Artnet and ddp temporarily

Module.h: add enabledChanged

SysModModel:
- setValueB: add rowNr parameter (tbd for other types): for rows: create array if not yet and assign value to array element

SysModModules:
- mdlEnabled.chFun: create value array if not exist, if exists set modules enabled status and trigger enabledChange function

SysModUI: 
- setChFunAndWs: add check for value arrays 
- processJson: check if var id has # (rowNr)
- processJson: if rowNr, use the value array to check the right value
- processJson: if bool call setValueB with rownr

SysModWeb:
- add print clIsFull.chFun 
- call setValueB with rowNr
- add addResponseArray to deal with value arrays

### CODE REVIEW

File: platformio.ini [hunk 1]
```diff
@@ -14,7 +14,9 @@ lib_deps =
 
 [appmod_leds]
 build_flags = 
-  -D APPMOD_LEDS -D USERMOD_ARTNET -D USERMOD_DDP
```
Comment (netmindz) [inline platformio.ini:17]: Why this change? Surely if we have the option to enable/disable at runtime why exclude these from the build? Especially as they have no library dependencies so have little impact of image size or compile time

File: src/Module.h [hunk 2]
```diff
@@ -35,6 +35,7 @@ class Module {
   virtual void loop() {}
 
   virtual void connected() {}
+  virtual void enabledChanged(bool tf) {}
```
Comment (netmindz) [inline src/Module.h:38]: If we have a module disabled before we connect WiFi, connect WiFi and then enable, the module will never get the connected call so might not function properly. Should we trigger a connected call for that module on enable or expect every module to handle this?

File: platformio.ini (hunk 1 above)
Comment (ewoudwijma) [reply platformio.ini:17]: Just temporary, as a consequence of me merging too soon in main 😉, if I run this I got all kinds of messages so I propose until the mods are working fine we leave it of in main.

File: src/Module.h (hunk 2 above)
Comment (ewoudwijma) [reply src/Module.h:38]: I think connected() and enabledChanged() are the 'orchestrators' and they can call functions which do the work. 

So function to connect can be called by both and if called by one of them, checks if connected is true and if enabled is true.

File: platformio.ini (hunk 1 above)
Comment (netmindz) [reply platformio.ini:17]: Maybe set the mod to be present, but disabled at runtime by default?
